hook-common/src/kafka_messages/app_metrics.rs

```diff
@@ -0,0 +1,123 @@
+use chrono::{DateTime, Utc};
+use serde::{Serialize, Serializer};
+use uuid::Uuid;
+
+use super::{serialize_datetime, serialize_uuid};
+
+#[derive(Serialize)]
+pub enum AppMetricCategory {
+    ProcessEvent,
+    OnEvent,
+    ScheduledTask,
+    Webhook,
+    ComposeWebhook,
+}
```

Comment:
I get that we are supporting all possible categories, but given that we would only ever output `Webhook` metrics, we could just have that as a const string?

Reply:
Sure, I don't mind that. My thinking is that we'll want a more general Rust library at some point (as in, I wish this type lived in some `posthog-rust-lib` repo instead), and I wanted my "port" of types to be faithful so I didn't miss anything.

Reply:
Yeah, I see. My original comment just imagined a lot of unnecessary `match` statements for only one enum variant. Do you think we should move these types to capture-rs and use that as a library as we discussed? I'll need to reference them from consumer, so wondering if you feel better with this being here in `hook-common` or in capture.

Reply:
Since capture doesn't need these, I feel like leaving them here is fine and we'll hopefully be able to combine repos at the offsite.